include/dolphin/gx/GXDispList.h

```diff
@@ -7,8 +7,15 @@
 extern "C" {
 #endif
 
+#ifdef AURORA
+#define GXCALLDISPLAYLISTLE GXCallDisplayListLE
```

Comment:
maybe `GXCallDisplayListNative`? No reason to make this all uppercase

Reply:
If we're going to do this here, might as well make the other PC  wrappers match.